target/linux/ath79/dts/qca9563_tplink_eap245-v3.dts

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	compatible = "tplink,eap245-v3", "qca,qca9563";
+	model = "TP-Link EAP245 v3";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_green;
+		led-failsafe = &led_orange;
+		led-running = &led_green;
+		led-upgrade = &led_orange;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_green: led_green {
+			label = "tp-link:green:power";
+			gpios = <&gpio 7 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_orange: led_orange {
+			label = "tp-link:orange:failsafe";
+			gpios = <&gpio 9 GPIO_ACTIVE_HIGH>;
+		};
+
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "Reset button";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&uart {
+	status = "okay";
+};
+
+&gpio {
+	status = "okay";
+};
+
+&spi {
+	status = "okay";
+	num-cs = <1>;
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <25000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "factory-boot";
+				reg = <0x000000 0x040000>;
+				read-only;
+			};
+
+			partition@40000 {
+				label = "u-boot";
+				reg = <0x040000 0x040000>;
+				read-only;
+			};
+
+			partition@80000 {
+				label = "partition-table";
+				reg = <0x080000 0x010000>;
+				read-only;
+			};
+
+			info: partition@90000 {
+				label = "info";
+				reg = <0x090000 0x010000>;
+				read-only;
+			};
+
+			art: partition@a0000 {
+				label = "art";
+				reg = <0x0a0000 0x010000>;
+				read-only;
+			};
+
+			partition@b0000 {
+				label = "extra-para";
+				reg = <0x0b0000 0x010000>;
+				read-only;
+			};
+
+			partition@c0000 {
+				compatible = "openwrt,okli";
+				label = "firmware";
+				reg = <0x0c0000 0xe40000>;
+			};
+
+			partition@f00000 {
+				label = "config";
+				reg = <0xf00000 0x030000>;
+				read-only;
+			};
+
+			partition@f30000 {
+				label = "mutil-log";
+				reg = <0xf30000 0x080000>;
+				read-only;
+			};
+
+			partition@fb0000 {
+				label = "oops";
+				reg = <0xfb0000 0x040000>;
+				read-only;
+			};
+
```

Comment:
drop empty line here.

Reply:
Also dropped another empty line after the led-block (IIRC)